src/Microsoft.AspNet.SignalR.Client/Transports/TransportHelper.cs

```diff
@@ -57,8 +57,6 @@ public virtual Task<string> GetStartResponse(IHttpClient httpClient, IConnection
 
             var startUrl = UrlBuilder.BuildStart(connection, transport, connectionData);
 
-            httpClient.Initialize(connection);
```

Comment:
To send the start request you need the connectionToken token you receive in the response to the negotiate request. Given that, this logic is hardcoded in the client and not something a user can control I think we can rely on the fact that negotiate will be called right before start. 
Initializing the httpClient when sending the start does not fully fix the issue since in this case the negotiate request will use a different http client than start request potentially resulting in sending requests to different servers and forcing separate https handshakes.